### PR TITLE
Removed initialization of empty VIRTUAL_ENV

### PR DESCRIPTION
**Pull request by buhl**
When spawning a new bash session when entering a virtual environment
the VIRTUAL_ENV is overwritten by this line.
However not having this line does not affect the bin/activate script.

### CODE REVIEW

File: themes/base.theme.bash [hunk 1]
```diff
@@ -74,7 +74,7 @@ THEME_SHOW_USER_HOST=${THEME_SHOW_USER_HOST:=false}
 USER_HOST_THEME_PROMPT_PREFIX=''
 USER_HOST_THEME_PROMPT_SUFFIX=''
 
-VIRTUAL_ENV=
+#VIRTUAL_ENV=
```
Comment (nwinkler) [inline themes/base.theme.bash:77]: Instead of commenting out, can you please remove this line (and the blank line after that)?

Comment (buhl) [reply themes/base.theme.bash:77]: Doh! Of course!

**Comment by buhl** [comment]
There you go :)
Sorry about the inconvenience.

**Comment by nwinkler** [comment]
Thanks!

**Pull request merged**